internal/namespaces/baremetal/v1alpha1/testdata/test-list-server-simple.stdout.golden

```diff
@@ -1,3 +1 @@
-ID                                    ORGANIZATION ID                       NAME                 DESCRIPTION  UPDATED AT       CREATED AT       STATUS  OFFER ID                              TAGS  IPS  DOMAIN                                                             BOOT TYPE  ZONE
-188b71df-4193-4f33-9151-df07234f3ef4  951df375-e094-4d26-97c1-ba548eeb9c42  bm-determined-ellis  -            few seconds ago  few seconds ago  ready   737f18c8-febc-4408-b69e-776bca0a3f48  [a]   2    188b71df-4193-4f33-9151-df07234f3ef4.fr-par-2.baremetal.scw.cloud  normal     fr-par-2
-eabed7e5-d03c-43f4-bc6d-e33f72fceb5b  951df375-e094-4d26-97c1-ba548eeb9c42  bm-sad-gauss         -            few seconds ago  few seconds ago  ready   964f9b38-577e-470f-a220-7d762f9e8672  [b]   2    eabed7e5-d03c-43f4-bc6d-e33f72fceb5b.fr-par-2.baremetal.scw.cloud  normal     fr-par-2
+-
```

Comment:
we should have at least a few servers in the list, to assert it works as intended 

remove file if unused